aws-s3/aws_s3.ml

```diff
@@ -2,3 +2,16 @@ module S3 = S3
 module Types = Types
 module Credentials = Credentials
 module Region = Region
+module Authorization = Authorization
+module Auth : sig val make_presigned_url :
```

Comment:
Why the new Auth module? The function is already exposed from the Authorization module.

I was trying to hide the functions in the Authorization module by not including it in Aws_s3 module, but I'm fine with including it. An alternative could be to create a module 'Presigned' (both .ml and .mli file) and only include that module in Aws_s3 if its sensible to hide the functions in the Authorization module

Reply:
Oops, this was actually a mistake, thanks, for catching it. I wasn't sure if you wanted to expose Authorization or just this function. If you are fine with exposing Authorization though, I'll switch it to that.

Reply:
I'm fine with exporting Authorization module, but I'm starting to move towards creating a new module to make it easier to find - But I can always do that after the PR has been merged - unless of course you have some comments on this.

Reply:
I don't have strong feelings either way.

Reply:
I ended up exporting the auth module but if you want to put it in it's own, that's fine with me.